packages/phpstan-rules/src/Rules/CheckOptionArgumentCommandRule.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\PHPStanRules\Rules;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\NodeFinder;
+use PhpParser\PrettyPrinter\Standard;
+use PHPStan\Analyser\Scope;
+use PHPStan\Type\ThisType;
+use Symfony\Component\Console\Command\Command;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Symplify\PHPStanRules\Tests\Rules\CheckOptionArgumentCommandRule\CheckOptionArgumentCommandRuleTest
+ */
+final class CheckOptionArgumentCommandRule extends AbstractSymplifyRule
+{
+    /**
+     * @var string
+     */
+    public const ERROR_MESSAGE = '%s() called in configure(), must be called with %s() in execute() in "Symfony\Component\Console\Command\Command" type';
+
+    /**
+     * @var string
+     */
+    private const METHOD_CALL_INVALID = [
+        'addOption' => 'getArgument',
+        'addArgument' => 'getOption',
+    ];
+
+    /**
+     * @var string
+     */
+    private const METHOD_CALL_VALID = [
+        'addOption' => 'getOption',
+        'addArgument' => 'getArgument',
+    ];
```

Comment:
this is declaration of method calls in configure vs in execute method, compared with execute which :

```
addOption() match with getOption()
addArgument() match with getArgument()
```

with same arg.